src/systems/acoustic_comms/AcousticComms.hh

```diff
@@ -76,6 +76,7 @@ namespace systems
   ///       * <seed> : Seed value to be used for random sampling.
   ///
   /// Here's an example:
+  /// ```
```

Comment:
where are the closing quotes?

Reply:
Good catch! That might have broken that page. Oops!! 0f65637